### PR TITLE
Adding new line to index.html

### PR DESCRIPTION
**Pull request by djeff06**
I add a new line in index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
     <title>Resourcify</title>
 </head>
 <body>
-   
+ Welcome to resourcify THis is djeff working on amin's project
```
Comment (medgues) [inline index.html:10]: header 3 tag messing <h3></h3>

**Review by medgues — CHANGES_REQUESTED** [review]
please change this

File: index.html [hunk 2]
```diff
@@ -7,6 +7,6 @@
     <title>Resourcify</title>
 </head>
 <body>
- Welcome to resourcify THis is djeff working on amin's project    
+ <H3> Welcome to resourcify THis is djeff working on amin's project <H3>
```
Comment (djeff06) [inline index.html:10]: Adding header 3

Comment (medgues) [reply index.html:10]: good job

**Comment by medgues** [comment]
it was good job djef

**Pull request merged**